ci/prow/make_config.go

```diff
@@ -109,13 +114,42 @@ type sectionGenerator func(string, string, yaml.MapSlice)
 // stringArrayFlag is the content of a multi-value flag.
 type stringArrayFlag []string
 
-var (
-	// Array constants used throughout the jobs.
-	allPresubmitTests = []string{"--all-tests", "--emit-metrics"}
-	releaseNightly    = []string{"--publish", "--tag-release"}
-	releaseLocal      = []string{"--nopublish", "--notag-release"}
+// ####################################################################################################
+// ############## data definitions that are used for the testgrid config file generation ##############
+// ####################################################################################################
+// baseTestgridTemplateData contains basic data about the testgrid config file.
+type baseTestgridTemplateData struct {
+	TestGroupName string
+	Year          int
+}
+
+// testGroupTemplateData contains data about a test group
+type testGroupTemplateData struct {
+	Base      baseTestgridTemplateData
+	GcsLogDir string
```

Comment:
This should be a base attribute. With Base being a baseProwJobTemplateData structure (and then renaming it to a more generic name would make more sense), it won' be necessary anymore. Please add a TODO.